### PR TITLE
[BLUEBUTTON-183] [BLUEBUTTON-184] Updated load testing strategy

### PR DESCRIPTION
**Pull request by rnagle**
Details to come...

### CODE REVIEW

File: locust/locustfile.py [hunk 1]
```diff
@@ -48,38 +41,33 @@ def check_count(self):
...
-    def get_patient(self):
-        resource ='/v1/fhir/Patient/%s' % self.patient
-        self.make_req(resource)
-
-    @task(6)
-    def get_metadata(self):
-        resource = '/v1/fhir/metadata'
+        patient = self.get_random_userinfo()
+        resource = '/v1/fhir/ExplanationOfBenefit/?patient=%s' % patient
         self.make_req(resource)
```
Comment (whytheplatypus) [inline locust/locustfile.py]: at the end of the sequence (which I think is here?) can you make a request to the `revoke-token` endpoint so the load test can kind of clean up after itself?

File: locust/locustfile.py [hunk 2]
```diff
@@ -48,38 +41,33 @@ def check_count(self):
...
-
-    @task(22)
-    def get_patient(self):
-        resource ='/v1/fhir/Patient/%s' % self.patient
-        self.make_req(resource)
-
-    @task(6)
-    def get_metadata(self):
-        resource = '/v1/fhir/metadata'
+        patient = self.get_random_userinfo()
```
Comment (whytheplatypus) [inline locust/locustfile.py]: the EOB endpoint shouldn't actually need this, it'll use the token to determine the patient, not a bad thing to  have though.

**Review by whytheplatypus — CHANGES_REQUESTED** [review]

File: locust/run.sh [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+if [ "$BB_LOAD_TEST_TYPE" != "eob" ] && [ "$BB_LOAD_TEST_TYPE" != "all" ]
+then
+  echo "Must specify a load test type (eob|all)" >&2
+  exit 1
+fi
+
+docker build -f ./Dockerfiles/Dockerfile.tkns -t bb_tkns .
+docker build -f ./Dockerfiles/Dockerfile.locust -t bb_locust .
+
+set +e
```
Comment (rnagle) [inline locust/run.sh]: Right now, `tkns` reliably fetches at least a few access tokens, but sometimes exits with code `> 0` due to a bug we haven't quite tracked down yet. I'm using `set +e` to ignore the exit code and let the script continue for the time being. We can/should tidy this up later.

**Review by whytheplatypus — APPROVED** [review]
lgtm

**Pull request merged**